soccer/planning/MotionInstant.hpp

```diff
@@ -27,4 +28,32 @@ struct MotionInstant {
     }
 };
 
+/**
+ * This class represents a robot's angle "state" at a given time, including
+ * angle and rotation velocity.
+ */
+struct AngleInstant {
+    explicit AngleInstant(boost::optional<float> angle = boost::none,
+                          boost::optional<float> angleVel = boost::none)
+        : angle(angle), angleVel(angleVel){};
+    boost::optional<float> angle;
```

Comment:
Consider making these non-optional since AngleInstant is already optional inside of RobotInstant